lib/rspec/support/differ.rb

```diff
@@ -73,6 +89,31 @@ def initialize(opts={})
 
     private
 
+      def hash_with_anything?(arg)
+        return false unless Hash === arg
+
+        @keys_with_anything = recursive_get_keys(arg)
+        @keys_with_anything.any?
+      end
+
+      def recursive_get_keys(hash)
+        klass = RSpec::Mocks::ArgumentMatchers::AnyArgMatcher
```

Comment:
What if this is not defined, like someone opted out of mocking completely, or uses rr/mocha etc?

Reply:
I think it’s ok to use ‘return [] unless defined?(RSpec::Mocks)’ here

Reply:
Nice! I didn't thought someone may opt out mocking completely. I've fixed it on my latest commit!